UndertaleModTool/MainWindow.xaml.cs

```diff
@@ -1349,7 +1349,7 @@ public void OpenCodeFile(string name, sbyte editorDecompile)
 
                 //TODO: find the way to scroll to the code item and highlight it.
                 
-                CodeEditorDecompile = editorDecompile;
+                CodeEditorDecompile = editorDecompile; //-1 - unstated, 0 - don't decompile, 1 - decompile
```

Comment:
Not sure about the full context here, but why isn't this an enum?

Reply:
I just didn't think about it.